src/test/java/org/testeditor/fixture/web/WebDriverFixtureLocalFirefoxTest.java

```diff
@@ -115,11 +115,11 @@ public void newFirefoxPortableStartAndStopTest() throws InterruptedException, IO
 		fixture.waitSeconds(2);
 		fixture.goToUrl(webSite.toString());
 		Assert.assertTrue(fixture.getDriver().getTitle().startsWith(expectedTitle));
-		fixture.typeInto(tool.getUserName(), "test_1");
-		String userName = fixture.getWebElement("[id]" + (tool.getUserName())).getAttribute("value");
+		fixture.typeInto(tool.getUserName(), LocatorStrategy.NAME, "test_1");
+		String userName = fixture.getWebElement(tool.getUserName(), LocatorStrategy.ID).getAttribute("value");
```

Comment:
yep, should do that with another ticket (which should include a better test coverage, too).